test/mocha/20-recovery.js

```diff
@@ -95,4 +95,38 @@ describe('Failure recovery', function() {
     const result2Hash = parseInt(result2CredentialStatusId.split('#')[1]);
     result2Hash.should.equal(result1Hash + 1);
   });
+
+  it('should error if credentialStatus.id does not exist', async () => {
+    const {integration: {secrets}} = agents;
+    const credential = helpers.cloneCredential();
+    credential.id = 'urn:someId1';
+    const {token} = secrets;
+
+    // Intentionally change the credentialStatus id when write is called
+    // the second time.
```

Comment:
Is this comment right? What's the purpose of this stub?

Reply:
I added the stub to force the count to be 0 when exists is being called, but it looks like it does that without the stub. So, the stub wasn't necessary.